From 0ab8e29eb84b224540ecc6e357b5d016fc61e78d Mon Sep 17 00:00:00 2001
From: Dan Pasanen <invisiblek@cyanogenmod.org>
Date: Sun, 13 Nov 2016 15:47:12 -0600
Subject: [PATCH 15/18] LatinIME: disable sound on keypress on all devices by
 default

* This sound is horrible and tablet users shouldn't be subject
  to listening to it through setupwizard either.

Change-Id: Ia6b6db31fb9a986c2a68773dc69524aed5e5af08
---
 java/res/values-sw600dp/config-per-form-factor.xml | 2 +-
 java/res/values-sw768dp/config-per-form-factor.xml | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/java/res/values-sw600dp/config-per-form-factor.xml b/java/res/values-sw600dp/config-per-form-factor.xml
index aa9a05482..bb2eaa85f 100644
--- a/java/res/values-sw600dp/config-per-form-factor.xml
+++ b/java/res/values-sw600dp/config-per-form-factor.xml
@@ -23,7 +23,7 @@
     <bool name="config_enable_show_key_preview_popup_option">false</bool>
     <!-- Whether or not Popup on key press is enabled by default -->
     <bool name="config_default_key_preview_popup">false</bool>
-    <bool name="config_default_sound_enabled">true</bool>
+    <bool name="config_default_sound_enabled">false</bool>
     <bool name="config_enable_show_voice_key_option">false</bool>
     <bool name="config_key_selection_by_dragging_finger">false</bool>
     <!-- Showing more keys keyboard, just above the touched point if true, aligned to the key if
diff --git a/java/res/values-sw768dp/config-per-form-factor.xml b/java/res/values-sw768dp/config-per-form-factor.xml
index b90fbaea5..45ff0b1ff 100644
--- a/java/res/values-sw768dp/config-per-form-factor.xml
+++ b/java/res/values-sw768dp/config-per-form-factor.xml
@@ -23,7 +23,7 @@
     <bool name="config_enable_show_key_preview_popup_option">false</bool>
     <!-- Whether or not Popup on key press is enabled by default -->
     <bool name="config_default_key_preview_popup">false</bool>
-    <bool name="config_default_sound_enabled">true</bool>
+    <bool name="config_default_sound_enabled">false</bool>
     <bool name="config_enable_show_voice_key_option">false</bool>
     <bool name="config_key_selection_by_dragging_finger">false</bool>
     <!-- Showing more keys keyboard, just above the touched point if true, aligned to the key if
-- 
2.17.1

